broker-node/app/Clients/BrokerNode.php

```diff
@@ -27,10 +28,13 @@ class BrokerNode
 
     public static $ChunkEventsRecord = null;
 
-    private static function initEventRecord()
+    private static $SegmentStarted = null;
+
+    private static function initSegment()
     {
-        if (is_null(self::$ChunkEventsRecord)) {
-            self::$ChunkEventsRecord = new ChunkEvents();
+        if (is_null(self::$SegmentStarted )) {
+            Segment::init("SrQ0wxvc7jp2XDjZiEJTrkLAo4FC2XdD");
```

Comment:
Shouldn't be committing this key into Github. If you add it to `broker-node/.env` as `SEGMENT_WRITE_KEY`

Reply:
I added it there, but could not get it to work.